pkgs/tools/X11/xborders/default.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  buildPythonPackage,
+  python3Packages,
+  fetchFromGitHub,
+  libwnck,
+  gtk3,
+  libnotify,
+  wrapGAppsHook,
+  gobject-introspection
+}:
```

Are you using [Alejandra](https://github.com/kamadorueda/alejandra)? 
According to https://github.com/NixOS/rfcs/pull/101, the blessed code formatter will be [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt). 

```suggestion
{ lib
  , buildPythonPackage
  , python3Packages
  , fetchFromGitHub
  , libwnck
  , gtk3
  , libnotify
  , wrapGappsHook
  , gobject-introspection
}:
```

---

Oh I didn't know that was a thing! I'll use it in future, thanks.